<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>co zjeść? Aplikacja</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>801</width>
      <height>511</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="main">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>405</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>wygeneruj przepis ze składników</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAll_Button">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/new/prefix1/all.png</normaloff>:/new/prefix1/all.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>118</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>205</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>dodaj nowy przepis</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAll_Button_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>270</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/new/prefix1/generate.png</normaloff>:/new/prefix1/generate.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>118</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>410</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>usuń przepisy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAll_Button_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>70</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/new/prefix1/add.png</normaloff>:/new/prefix1/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>118</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAll_Button_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/new/prefix1/delete.png</normaloff>:/new/prefix1/delete.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>118</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>wyświetl wszystkie przepisy</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>491</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>799</width>
         <height>489</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dodaj nowy przepis</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recipeName">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="addIngredient">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>170</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj składnik</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveRecipe">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>460</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Zapisz przepis</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_42">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Light</family>
        <pointsize>9</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nazwa przepisu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_43">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Light</family>
        <pointsize>9</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Składniki</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ingredient">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Light</family>
        <pointsize>9</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Dodane składniki:</string>
      </property>
     </widget>
     <widget class="QLabel" name="addedIngredients">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>741</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Light</family>
        <pointsize>9</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_45">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Light</family>
        <pointsize>9</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Przepis</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="recipe">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>751</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>491</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>799</width>
         <height>489</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Znajdź swój przepis</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_46">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Light</family>
        <pointsize>9</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Składniki</string>
      </property>
     </widget>
     <widget class="QWidget" name="central" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>621</width>
        <height>81</height>
       </rect>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>621</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout"/>
      </widget>
     </widget>
     <widget class="QPushButton" name="filterButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>180</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Filtruj</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Znalezione przepisy</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="foundRecipesArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>801</width>
        <height>211</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>799</width>
         <height>209</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="zasoby.qrc">:/new/prefix1/logo.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
